application/common/mail/password-changed.html.php

```diff
@@ -0,0 +1,94 @@
+<?php
+use common\helpers\MySqlDateTime;
+use yii\helpers\Html as yHtml;
+
+/**
+ * @var string $employeeId
+ * @var string $firstName
+ * @var string $lastName
+ * @var string $displayName
+ * @var string $username
+ * @var string $email
+ * @var string $active
+ * @var string $locked
+ * @var string $lastChangedUtc
+ * @var string $lastSyncedUtc
+ * @var string $lastLoginUtc
+ * @var string $passwordExpiresUtc
+ * @var string $emailSignature
+ * @var string $helpCenterUrl
+ * @var string $idpDisplayName
+ * @var string $passwordForgotUrl
+ * @var string $passwordProfileUrl
+ * @var string $supportEmail
+ * @var string $supportName
+ * @var bool   $isMfaEnabled
+ */
+?>
+<p>
+    Dear <?=yHtml::encode($displayName)?>,
+</p>
+<p>
+    The password for your <?=yHtml::encode($idpDisplayName)?> account has been changed. If you did not make this change
+    please contact <?=yHtml::encode($supportName)?> at <?=yHtml::encode($supportEmail)?> immediately to let us know.
+</p>
+<p>
+    Please remember that this account will be your primary means for logging into many corporate
+    applications. It is also important to note that this account (username and password) is unique and will not be kept
+    in sync with any other accounts you have.
+</p>
+<p>
+    Password changed on: <?=yHtml::encode(MySqlDateTime::formatDateForHumans($lastChangedUtc))?><br />
+    Password expires on: <?=yHtml::encode(MySqlDateTime::formatDateForHumans($passwordExpiresUtc))?>
```

Comment:
What would you think of converting the date to this human-friendly format in `User->getAttributesForEmail()` rather than in the view, so that we don't have to do this in every view that needs it?

Reply:
Oh yeah, that could be nice, I was thinking I couldn't update it further upstream because it was the `fields()` format for other uses. I can move it there.

Reply:
fixed